File: API/Entities/Member.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entities
{
    [Table("Member")]
    public class Member
    {
        public int MemberId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string CellPhone { get; set; }
        public string Gender { get; set; }
        public bool Scout { get; set; }
        public string CurrentPosition { get; set; }
        public string CurrentRank { get; set; }
        public string Photo { get; set; }
        public string Patrol { get; set; }
        public int PatrolId { get; set; }
        public int TroopNumber { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public DateTime? RechartedDate { get; set; }

        public ICollection<Transaction> Transactions { get; set; }
    }
}

File: client/src/app/_models/member.ts
import { Transaction } from "./transaction";

export interface Member {
  memberId: number;
  firstName: string;
  lastName: string;
  email: string;
  cellPhone: string;
  gender: string;
  scout: boolean;
  currentPosition: string;
  currentRank: string;
  photo: string;
  patrol: string;
  patrolId: number;
  troopNumber: number;
  age: number;
  rechartedDate?: Date;
  transactions: Transaction[];
}
